lerna.json

```diff
@@ -16,6 +16,7 @@
     "src/components/20-molecules/datepicker",
```

Comment:
- knobs are being used
- I think this is the only option we have, because you can upload images with all varieties of height to length ratio. And all images should have the same size on the overview
- yes there is a default max. I'm still waiting for the default values Bettina will give me from the old file-upload
- gonna fix that react example
- good point

Reply:
Important question from me: I have set the maxSizes for file in KB. So the maxSizes are for the compromised files. Do you think it should be for the not-compromised files?

Reply:
maxSizes  is for the compressed ones as that is the relevant for the server. max 10 files default comes from AEM forms.

Reply:
Gut dann sind wir uns einig :)